autoscale/src/test/java/com/sequenceiq/periscope/converter/DistroXAutoscaleScalingActivityResponseConverterTest.java

```diff
@@ -0,0 +1,70 @@
+package com.sequenceiq.periscope.converter;
+
+import static java.time.temporal.ChronoUnit.MINUTES;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Instant;
+import java.util.Date;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.StackType;
+import com.sequenceiq.periscope.api.model.ActivityStatus;
+import com.sequenceiq.periscope.api.model.ClusterState;
+import com.sequenceiq.periscope.api.model.DistroXAutoscaleScalingActivityResponse;
+import com.sequenceiq.periscope.domain.Cluster;
+import com.sequenceiq.periscope.domain.ClusterPertain;
+import com.sequenceiq.periscope.domain.ScalingActivity;
+
+public class DistroXAutoscaleScalingActivityResponseConverterTest {
```

Comment:
This is missing `@ExtendWith(MockitoExtension.class)`. Probably why you had to explicitly inject your class under test.

Reply:
Changed